extension/scripts/global/featureManager.js

```diff
@@ -13,6 +14,10 @@ class FeatureManager {
 			this.features.forEach((feature) => this.executeFunction(feature.cleanup));
 			this.isDisconnected = true;
 		});
+		this.log = async (params, error) => {

```

Comment:
Why would you replace `console.log` with this? `console.log` tells me where it will be logged at least.